File: Blagajna/src/app/den-bl/den-izvod.model.ts
import { Doc } from "./doc.model"

export class DenIzvod {
    public id?: number
    public izvodDate: string
    public vkupnaIsplata: number
    public vkupenPriem: number
    public saldo: number
    public denBlSostojba: number
    public finalIzvod: boolean
    public denDocuments?: Doc[]

     constructor( izvodDate: string, vkupnaIsplata: number, vkupenPriem: number,
    saldo: number, denBlSostojba: number, finalIzvod: boolean) {
         
        this.izvodDate = izvodDate;
        this.vkupnaIsplata = vkupnaIsplata;
        this.vkupenPriem = vkupenPriem;
        this.saldo = saldo;
        this.denBlSostojba = denBlSostojba;
        this.finalIzvod = finalIzvod

        
     }
}

File: Blagajna API/Blagajna.Api/Blagajna.Models/DenIzvodModel.cs
using Blagajna.Domain.Den.Bl.Models;
using System;
using System.Collections.Generic;
using System.Text;

namespace Blagajna.Models
{
    public class DenIzvodModel
    {
        public int Id { get; set; }
        public DateTime IzvodDate { get; set; }
        public int VkupnaIsplata { get; set; }
        public int VkupenPriem { get; set; }
        public int Saldo { get; set; }
        public int DenBlSostojba { get; set; }
        public bool FinalIzvod { get; set; }
        public List<DenDocumentModel> DenDocuments { get; set; }
    }
}
